.babelrc

```diff
@@ -1,27 +1,16 @@
 {
+  "presets": [["env", {
+    "targets": {
+      "node": 4
+    }
+  }]],
   "plugins": [
-    "syntax-async-functions",
     "syntax-async-generators",
     "transform-class-properties",
-    "transform-flow-strip-types",
-    "transform-object-rest-spread",
-    "transform-es2015-template-literals",
-    "transform-es2015-literals",
-    "transform-es2015-function-name",
-    "transform-es2015-arrow-functions",
-    "transform-es2015-block-scoped-functions",
-    ["transform-es2015-classes", {"loose": true}],
-    "transform-es2015-object-super",
-    "transform-es2015-shorthand-properties",
-    "transform-es2015-duplicate-keys",
-    "transform-es2015-computed-properties",
-    "check-es2015-constants",
-    ["transform-es2015-spread", {"loose": true}],
-    "transform-es2015-parameters",
     ["transform-es2015-destructuring", {"loose": true}],
-    "transform-es2015-block-scoping",
-    "transform-es2015-modules-commonjs",
-    "transform-regenerator",
-    "transform-es3-property-literals"
+    ["transform-es2015-spread", {"loose": true}],
```

Comment:
This was defined twice. I kept the `loose` mode dep but if `loose` can be turned off, both this and the destructuring transform could be removed since they are already applied in the Node4 preset.